packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -31,15 +32,33 @@ class NodeMessageBase(BaseModel):
 class LoggerRabbitMessage(RabbitMessageBase, NodeMessageBase):
     channel_name: Literal["simcore.services.logs"] = "simcore.services.logs"
     messages: list[str]
+    log_level: int = logging.INFO
 
 
 class EventRabbitMessage(RabbitMessageBase, NodeMessageBase):
     channel_name: Literal["simcore.services.events"] = "simcore.services.events"
     action: RabbitEventMessageType
 
 
+class ProgressType(Enum):
+    COMPUTATION_RUNNING = auto()  # NOTE: this is the original only progress report
```

Comment:
``auto`` in this context will produce integers while you have defined
```python
"progress_type": str,
```
In addition, if this has to be identified by the front-end, it can be error-prone with integers since it depends on the order in which you write the items. I suggest using auto such that it generaates a value identical to the name. 
Here is an example from the documentation where I inherited ``str``  as well
```python
class AutoName(str, Enum):
    def _generate_next_value_(name, start, count, last_values):
        return name

class Ordinal(AutoName):
    NORTH = auto()
    SOUTH = auto()
    EAST = auto()
    WEST = auto()


def test_autoname():
    assert list(f"{n}" for n in Ordinal) == [
        "NORTH",
        "SOUTH",
        "EAST",
        "WEST",
    ]
```

Reply:
I pass .name to the frontend. but anyway I will add this

Reply:
done